src/common/sdk/web/_connectedAdapterName.mdx

```diff
@@ -0,0 +1,11 @@
+You can configure a lot of [adapters](/sdk/web/adapters) in the Web3Auth instance. If you have installed one of the adapters, you can also use the
+`connectedAdapterName` to get the currently used adapter.
+
+```javascript
+// Initialized Web3Auth instance
+
+// User is logged in using one of the adapters (metamask, torus-evm, etc.)
+
+const adapterName = web3auth.connectedAdapterName;
+// returns the currently used adapter name (metamask, torus-evm, etc.)
```

Comment:
it is not a function, doesn't return the adapter name. Change this to be - contains the name of the adapter currently in use.